hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -693,6 +712,9 @@ public GroupProperties(Config config) {
         VERSION_CHECK_ENABLED = new GroupProperty(config, PROP_VERSION_CHECK_ENABLED, "true");
         PREFER_IPv4_STACK = new GroupProperty(config, PROP_PREFER_IPv4_STACK, "true");
         IO_THREAD_COUNT = new GroupProperty(config, PROP_IO_THREAD_COUNT, "3");
+        IO_INPUT_THREAD_COUNT = new GroupProperty(config, PROP_IO_INPUT_THREAD_COUNT, "-1");
```

Comment:
Why not just?: 
`IO_INPUT_THREAD_COUNT = new GroupProperty(config, PROP_IO_INPUT_THREAD_COUNT, IO_THREAD_COUNT)`

This way its default value will be value of `IO_THREAD_COUNT`.